kie-wb-parent/kie-wb-webapp/src/main/resources/security-module-process-admin.properties

```diff
@@ -19,6 +19,7 @@ role.process-admin.permission.perspective.read.UserHomePagePerspective=true
 role.process-admin.permission.perspective.read.ProcessDefinitions=true
 role.process-admin.permission.perspective.read.ProcessInstances=true
 role.process-admin.permission.perspective.read.Tasks=true
+role.process-admin.permission.perspective.read.TaskAdmin=true
```

Comment:
I can't reproduce the issue. How are you running the workbench? In my case using `kie-wb-webapp.war` generated in `kie-wb-distributions` it works correctly (the Task Administration option isn't present for regular users)